great_expectations/profiler/domain_builder/simple_semantic_type_domain_builder.py

```diff
@@ -55,13 +56,17 @@ def _get_domains(
         else:
             # TODO: <Alex>ALEX -- We should make this error message more informative.</Alex>
             raise ValueError("unrecognized")
-        columns = validator.get_metric(MetricConfiguration("table.columns", dict()))
-        domains = []
+        domains: List[Dict[str, Union[str, MetricDomainTypes, Dict[str, Any]]]] = []
+        columns: List[str] = validator.get_metric(
+            MetricConfiguration("table.columns", {})
+        )
+        column: str
         for column in columns:
             # TODO: <Alex>ALEX -- Is this "column_type"?  Or is this more of a "domain type" (or "the name of the domain type")?</Alex>
```

Comment:
I believe this is a semantic type, which is distinguished from the column storage type. A storage type would be something like `int`. The semantic type would be something like `id`.

Reply:
🙇 Changing the code to reflect this clarification.  Thanks!